packages/origin-backend/src/pods/user/user.controller.ts

```diff
@@ -73,4 +75,22 @@ export class UserController {
     public async get(@Param('id') id: string) {
         return this.userService.findById(id);
     }
+
+    @Put('profile/:id')
+    @UseGuards(AuthGuard('jwt'))
+    public async putProfile(@Param('id') id: string, @Body() body: IUser) {
+        return this.userService.updateProfile(id, body);
+    }
+
+    @Put('password')
+    @UseGuards(AuthGuard('jwt'))
+    public async putChainAddress(@Body() body: UserPasswordUpdate) {
```

Comment:
The user should only be able to update his own password (or perhaps OrganizationAdmin can also do that?).
Anyway, this requires proper auth.

Also, wondering why the method is called `putChainAddress()` but it updates the user password.